.copr/yggdrasil.spec.in

```diff
@@ -0,0 +1,80 @@
+%bcond_without check
+
+# https://github.com/redhatinsights/yggdrasil
+%global goipath         github.com/redhatinsights/yggdrasil
+%global commit          @COMMIT@
+
+%gometa -f -i
+
+# Manually redefine %%dist to work around an issue in COPR where the build root
+# that creates the srpm does not define a value for %%dist. This should *NOT* be
+# carried in downstream; this is strictly an upstream/COPR/CI workaround.
+%global dist %{distprefix}.fc%{fedora}
+
+%global common_description %{expand:
+yggdrasil is a system daemon that subscribes to topics on an MQTT broker and
+routes any data received on the topics to an appropriate child "worker" process,
+exchanging data with its worker processes through a D-Bus message broker.}
+
+%global golicenses      LICENSE
+%global godocs          CONTRIBUTING.md README.md
+
+Name:           yggdrasil
+Version:        0.3.1
+Release:        %autorelease -p
+Summary:        Remote data transmission and processing client
+
+License:        GPL-3.0-only
+URL:            %{gourl}
+Source:         %{gosource}
+
+BuildRequires:  systemd-rpm-macros
+BuildRequires:  meson
+BuildRequires:  pkgconfig(dbus-1)
+BuildRequires:  pkgconfig(systemd)
+BuildRequires:  pkgconfig(bash-completion)
+
+%description %{common_description}
+
+%gopkg
+
+%prep
+%goprep
+%autopatch -p1
```

Comment:
Is this necessary? There is not `Patch` in this spec file.

Reply:
This is drift from the fact that I lifted [the downstream spec file](https://src.fedoraproject.org/rpms/yggdrasil/blob/rawhide/f/yggdrasil.spec) and modified it to work under these conditions. In a specfile intended to be built exclusively by CI automation, you are correct; there will probably never be manual patches that need to be applied.

Reply:
OK